dev-libs/opensp/opensp-1.5.2.recipe

```diff
@@ -0,0 +1,115 @@
+SUMMARY="A C++ library for using SGML/XML files"
+DESCRIPTION="An SGML Extended Facilities system conforming to Annex A of International Standard ISO/IEC 10744 -- Hypermedia/Time-based Structuring Language"
+HOMEPAGE="https://openjade.sourceforge.net/"
+COPYRIGHT="1994, 1995, 1996, 1997, 1998 James Clark"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://downloads.sourceforge.net/openjade/OpenSP-$portVersion.tar.gz"
+CHECKSUM_SHA256="57f4898498a368918b0d49c826aa434bb5b703d2c3b169beb348016ab25617ce"
+SOURCE_DIR="OpenSP-$portVersion"
+PATCHES="opensp-1.5.2.patchset"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+commandSuffix=$secondaryArchSuffix
+commandBinDir=$binDir
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	commandSuffix=
+	commandBinDir=$prefix/bin
+fi
+
+libVersion="5.0.0"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
+PROVIDES="
+	opensp$secondaryArchSuffix = $portVersion
+	lib:libosp$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	opensp${secondaryArchSuffix}_devel = $portVersion
+	devel:libosp$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES_devel="
+	opensp$secondaryArchSuffix == $portVersion base
+	"
+
+SUMMARY_tools="Command line tools for opensp"
+PROVIDES_tools="
+	opensp${secondaryArchSuffix}_tools = $portVersion
+	cmd:onsgmls$commandSuffix
+	cmd:osgmlnorm$commandSuffix
+	cmd:ospam$commandSuffix
+	cmd:ospcat$commandSuffix
+	cmd:ospent$commandSuffix
+	cmd:osx$commandSuffix
+	"
+REQUIRES_tools="
+	haiku$secondaryArchSuffix
+	opensp$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:automake
+	cmd:autoreconf
+	cmd:gcc$secondaryArchSuffix
+	cmd:gettext
+	cmd:libtoolize$secondaryArchSuffix
+	cmd:make
+	cmd:perl
+	"
+
+BUILD()
+{
+	autoreconf -if
+	runConfigure --omit-dirs "binDir" ./configure \
+		--bindir=$commandBinDir \
+		--disable-doc-build \
+		--disable-static
+	make $jobArgs
+}
+
+INSTALL()
+{
+	make install
+
+	rm -f $libDir/lib*.la
+
+	mkdir -p $docDir/tools
+	mkdir -p $developDocDir
+	cd $prefix/data/doc/OpenSP/doc
+	mv nsgmls.htm \
+		sgmlnorm.htm \
+		spam.htm \
+		spcat.htm \
+		spent.htm \
+		sx.htm \
+		$docDir/tools
+	mv $prefix/data/doc/OpenSP/doc $developDocDir
+	mv $prefix/data/doc* $docDir
+	rm -rf $prefix/data/doc
```

Comment:
I would replace `$prefix/data` with `$dataDir`.

Also, you use `cd` into a directory which is later deleted. Not sure if this can cause problems. Maybe replace it with `pushd` and add `popd` after the `mv`.